src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -548,16 +558,23 @@ class ObservationStore(
       plotIds: Collection<MonitoringPlotId>,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +556]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->It may make more sense to move MONITORING_PLOTS.IS_AD_HOC to OBSERVATION_PLOTS, so we can be consistent about MONITORING_PLOTS holding geometry and OBSERVATION_PLOTS holding observation characteristics<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2687?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
It's a property of the plot itself too, though: ad-hoc plots should be completely ignored by all the code that assigns plots for regular observations, even if they fall within the site boundary.